test/dlc_tests/container_tests/bin/testMXNetHVD

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-INSTANCE_TYPE=`curl http://169.254.169.254/latest/meta-data/instance-type`
+INSTANCE_TYPE=$1
```

Comment:
Can we completely remove the `INSTANCE_TYPE` from the script. We can handle the skipping the test based on the instance type using pytest fixtures.